src/ducks/balance/Balance.jsx

```diff
@@ -380,10 +381,19 @@ class Balance extends React.Component {
         {flag('balance-history') ? (
           <History
             accounts={historyData['io.cozy.bank.accounts']}
-            transactions={historyData['io.cozy.bank.operations']}
             chartProps={{
               data: chartData,
-              width: chartNbTicks * chartIntervalBetweenPoints
+              width: chartNbTicks * chartIntervalBetweenPoints,
+              height: 103,
+              margin: {
+                top: 10,
+                bottom: 35,
+                left: 16,
+                right: 16
+              },
+              showAxis: true,
+              axisMargin: 10,
+              tickFormat: d3.timeFormat('%b')
```

Comment:
Minor: I would extract the d3.timeFormat outside the component

Reply:
I don't understand where you would put it. Can you tell me more ?

Reply:
sure :) at the top of the component

```
const TICK_FORMAT = d3.timeFormat('%b')

tickFormat: TICK_FORMAT
```